rosidl_typesupport_c/src/type_support_dispatch.hpp

```diff
@@ -44,15 +41,24 @@ get_typesupport_handle_function(
     return handle;
   }
 
-#ifdef ROSIDL_TYPESUPPORT_C_USE_POCO
   if (handle->typesupport_identifier == rosidl_typesupport_c__typesupport_identifier) {
     const type_support_map_t * map = \
       static_cast<const type_support_map_t *>(handle->data);
     for (size_t i = 0; i < map->size; ++i) {
       if (strcmp(map->typesupport_identifier[i], identifier) != 0) {
         continue;
       }
-      Poco::SharedLibrary * lib = nullptr;
+      rcutils_shared_library_t * lib = nullptr;
+      rcutils_allocator_t allocator = rcutils_get_default_allocator();
+      lib = static_cast<rcutils_shared_library_t *>(allocator.allocate(
+              sizeof(rcutils_shared_library_t), allocator.state));
```

Comment:
This indent looks wrong, it should never be more than two spaces per indent:

```suggestion
        sizeof(rcutils_shared_library_t), allocator.state));
```

Reply:
fixed